### PR TITLE
Housekeeping: add link to `"exports"` PR

### PR DESCRIPTION
**Pull request by GeoffreyBooth**

### CODE REVIEW

File: doc/plan-for-new-modules-implementation.md [hunk 1]
```diff
@@ -101,7 +101,8 @@ Phase 3 improves user experience and extends the MVP. Phase 3 is malleable based
 
 * `"exports"` field: for consumers of a package, map the paths of deep imports to provide encapsulation (an explicit public API); pretty specifiers (no file exensions or paths that include things like `dist/`) and flexibility for future package versions renaming or moving files without affecting the package’s public API. Applies to both ESM and CommonJS.
   - Proposal: [Package Exports Proposal](https://github.com/jkrems/proposal-pkg-exports).
-  - **Status**: Ready for PR.
+  - Upstream PR: https://github.com/nodejs/node/pull/28568.
```
Comment (ljharb) [inline doc/plan-for-new-modules-implementation.md]:
```suggestion
  - Partial upstream PR: https://github.com/nodejs/node/pull/28568.
```
?

**Comment by GeoffreyBooth** [comment]
I don’t know if this was discussed in the meeting, I assume this is uncontroversial and we can just merge it in?

**Comment by jkrems** [comment]
:shipit:

**Comment by jkrems** [comment]
I think that's a yes.

**Pull request merged**